csharp/ql/lib/ext/generated/System.Collections.model.yml

```diff
@@ -41,11 +41,16 @@ extensions:
       - ["System.Collections", "Hashtable", False, "Synchronized", "(System.Collections.Hashtable)", "", "Argument[0]", "ReturnValue.SyntheticField[System.Collections.Hashtable+SyncHashtable._table]", "value", "dfc-generated"]
       - ["System.Collections", "Hashtable", False, "get_EqualityComparer", "()", "", "Argument[this].SyntheticField[System.Collections.Hashtable._keycomparer]", "ReturnValue", "value", "dfc-generated"]
       - ["System.Collections", "ICollection", True, "get_SyncRoot", "()", "", "Argument[this]", "ReturnValue", "value", "dfc-generated"]
+      - ["System.Collections", "IComparer", True, "Compare", "(System.Object,System.Object)", "", "Argument[0]", "Argument[this]", "taint", "df-generated"]
+      - ["System.Collections", "IComparer", True, "Compare", "(System.Object,System.Object)", "", "Argument[1]", "Argument[this]", "taint", "df-generated"]
```

Comment:
The same comment applies as with `System.Collections.IEqualityComparer`

Reply:
I haven't checked this concretely, but with very high probability it is the same issue as mentioned above.